docs/internationalization_translation.md

```diff
@@ -39,3 +39,13 @@ Native Name: Deutsch
 2. Add the language tag and native name in `src/translations/translationMetadata.json`.  Examples: "Français", "Français (CA)"
 3. Add the new language in Lokalize.
 Note: Sometimes you have to change the tag in Lokalise (Language -> Language settings -> custom ISO code).
+
+## Language specific guidelines
+Most languages have multiple possible translations of a sentence. Please take a look at the guidelines for your language here, where you can also find some typical mistakes to prevent.
+The sections are written in their corresponding languages, as this makes explaining the grammar easier and only native speakers should submit translations (see [Rules](#rules)).
+
+### German
+- Du/Sie: Dutze in den Übersetzungen, und verwende nicht das formale "Sie".
```

Comment:
Using the formal or the informal approach always leads to discussions. Most big Open Source projects are using the formal approach and we should stick to that. With the formal approach we follow the defacto standard for the localization of software and documentation.

For TTS and voice assistance features I'm all for informal because this lowers the barrier.